doc/lua_api.txt

```diff
@@ -6546,7 +6548,7 @@ Used by `minetest.register_node`.
                 },
                 {
                     -- Only drop if using a tool whose name is identical to one
-                    -- of these.
+                    -- of these (this tool filtering is deprecated).
```

Comment:
As far as I can tell, Wuzzy only suggested removing the string matching (below), not the list of tools.
I think it's useful to keep this one even when group matching is eventually implemented.

Reply:
Removed deprecation comment from this section.